btree/btree.c

```diff
@@ -4572,11 +4569,11 @@ static void vkvv_child(struct slot *slot, struct segaddr *addr)
 	int               total_size     = 1ULL << h->vkvv_shift;
 	void             *start_val_addr = (void*)h + total_size;
 	int               index          = slot->s_idx;
-	uint16_t          vspace         = vkvv_get_vspace();
+	uint32_t          vspace         = vkvv_get_vspace();
 	void             *new_addr;
 
 	if (h->vkvv_level > 0) {
```

Comment:
No need for this check.

Reply:
I have removed it but I may bring it in future if required for multi threaded UTs